### DISCUSSION TITLE
有没有什么设置可以隐藏文件夹（不隐藏子文件夹），但是不影响访问？

### DISCUSSION THREAD

**Discussion by tinyhugh** [AlistGo/alist#6320] [General]
有没有什么设置可以隐藏文件夹（不隐藏子文件夹），但是不影响访问，可以通过链接直接访问，只是看不到而已，不能用元信息隐藏，用元信息添加后还需要密码才能访问。

**Comment by wulicjk** [comment]
知道了告诉我下，我想隐藏文件夹，但是进入到这个文件夹目录时，还是能看到其中的子文件。但是目前好像看不到其中的子文件

**Reply by tinyhugh** [reply]
填一个元信息，路径为/，在隐藏里填写你想隐藏的文件夹，不勾选应用到子文件夹。
这样有个问题就是只能访问子文件夹内的子文件夹，比如我设置隐藏文件夹a，a里面是a1、a2……，只能通过/a/a1访问，不能通过/a访问看到a1、a2……。

**Reply by verydemo** [reply]
可以用不同的用户， 设置不同的访问权限

**Reply by tinyhugh** [reply]
> 可以用不同的用户， 设置不同的访问权限

并没有找到有相关设置可以设置某个文件夹对于某个用户的权限。
